pkg/config/data/config_schema_v1.0.json

```diff
@@ -161,6 +184,19 @@
           "description": "The default target for number of concurrent predictions. This setting can be used by an autoscaler to determine when to scale a deployment of a model up or down."
         }
       }
+    },
+    "tests": {
+      "$id": "#/properties/build/properties/tests",
```

Comment:
This `$id` still doesn't make sense to me 😅 In the test yaml above, `tests` is at the same level as `build`, but this `$id` indicates `tests` is a sub-key of `build`. I expected the value to be `#/properties/tests`.

Reply:
Resolved in [aa83e8b](https://github.com/replicate/cog/pull/2149/commits/aa83e8bbb0d70c2dc2bd8184e8982a13e827189b)